app/utils/tests/constants.test.js

```diff
@@ -0,0 +1,12 @@
+import { RESTART_ON_REMOUNT, DAEMON, ONCE_TILL_UNMOUNT } from '../constants';
```

Comment:
we can delete the constants.js its not used anywhere

Reply:
Done